jbtearlyaccesstarget/multiple/jbtearlyaccess-multiple.target

```diff
@@ -11,6 +11,12 @@
       <unit id="angularjs-eclipse-feature.feature.group" version="1.1.0.201511091212"/>
     </location>
 
+    <!-- Windup Migration -->
+    <location includeAllPlatforms="false" includeMode="slicer" includeSource="false" type="InstallableUnit">
+      <repository location="http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/"/>
```

Comment:
By linking to a SPECIFIC version of Windup toolkit here, we will end up pulling a SPECIFIC version into the Early Access update site... and when a new version drops, you'll have to rebuild the Early Access Central site to fetch the latest.

Locking down the version in a targetplatform definition is good from a stability perspective, but if your updates are not aligned to the 3week cadence of jbosstools/devstudio milestone releases, you might want to omit this and simply link to the http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/ site in the plugin.xml files instead.